content/examples/prometheus/envoy-alerts.yml

```diff
@@ -0,0 +1,295 @@
+groups:
+  - name: envoy-listeners
+    rules:
+      - alert: ListenerConnectionOverflow
+        expr: rate(envoy_listener_downstream_cx_overflow[1m]) > 0
+        for: 1m
+        labels:
+          severity: critical
+          component: envoy
+        annotations:
+          summary: "Listener connection limit exceeded"
+          description: "{{ $value }} connections/sec rejected at listener {{ $labels.envoy_listener_address }}"
+          runbook: |
+            1. Check Envoy resource usage (CPU, memory, file descriptors)
+            2. Review listener connection limits configuration
```

Comment:
Add a link to docs on this so operators can jump straight to the relevant configurations.